index.php

```diff
@@ -21,11 +21,11 @@
 
 //		Add breadcrumbs here
 $pagesArray = array(
```

Comment:
I'm not sure if this is the best solution for that. Let's have a quick discussion about it.